scipy/optimize/_optimize.py

```diff
@@ -2742,6 +2750,7 @@ def bracket(func, xa=0.0, xb=1.0, args=(), grow_limit=110.0, maxiter=1000):
     This function can find a downward convex region of a function:
 
     >>> import matplotlib.pyplot as plt
+    >>> import numpy as np
```

Comment:
"import numpy as np" should be first than "import matplotlib.pyplot as plt".

Reply:
I'll take note of this.